### PR TITLE
Made the Roof Solars Start Wired

### PR DESCRIPTION
**Pull request by Geevies**
I tried my best but couldn't get them to start connected as well, so if you want to get the solars running, all you have to do is access the computer and click the "Search for Connected" button. It'll find the solars and arrange them, you have to do nothing else.

If anyone can figure out why this isn't working, you get a metaphorical cookie.
image

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
Roof solars work automatically now. No need for crew / AI intervention.

File: code/modules/power/solar.dm [hunk 1]
```diff
@@ -528,11 +528,15 @@
 	track = 2 // Auto tracking mode
 
 /obj/machinery/power/solar_control/autostart/Initialize()
-	. = ..()
-	src.search_for_connected()
+	..()
+	do_solars()
```
Comment (NonQueueingMatt) [inline code/modules/power/solar.dm]: This can 100% be removed without affecting anything.

File: code/modules/power/solar.dm [hunk 2]
```diff
@@ -528,11 +528,15 @@
...
+	addtimer(CALLBACK(src, .proc/do_solars), 1800) // do it again three minutes later, just to be sure
```
Comment (NonQueueingMatt) [inline code/modules/power/solar.dm]: I'm pretty sure you can handle this in LateInitialize, you just need to check our wiki on how to actually do post-mapload late init.

Comment (Geevies) [reply code/modules/power/solar.dm]: I'm not keen on it, I barely got this working as-is. I also kinda like that the solars only kick in some  time after the shift starts.

File: code/modules/power/solar.dm [hunk 3]
```diff
@@ -528,11 +528,14 @@
 	track = 2 // Auto tracking mode
 
 /obj/machinery/power/solar_control/autostart/Initialize()
-	. = ..()
-	src.search_for_connected()
+	..()
```
Comment (NonQueueingMatt) [inline code/modules/power/solar.dm]:
```suggestion
	. = ..()
```

**Pull request merged**